cliStocksTracker.py

```diff
@@ -298,31 +229,31 @@ def average_buyin(self, buys: list, sells: list):
 
         return count, bought_at
 
+    # download all ticker data in a single request
+    # harder to parse but this provides a signficant performance boost
+    def download_market_data(self, args, stocks):
+        # get graph time interval and period
+        time_period = args.time_period if args.time_period else "1d"
+        time_interval = args.time_interval if args.time_interval else "1m"
+
+        return market.download(
+            tickers=stocks,
+            period=time_period,
+            interval=time_interval,
+            progress = False
+        )
```

Comment:
single download for all ticker data. will wrap this in a try catch per the other PR that was just merged